src/main.css

```diff
@@ -68,6 +68,10 @@ code {
 	hyphens: none;
 }
 
+pre {
```

Comment:
can this go on a `code[class*='language-']` selector? I'm trying to remove as many generic tag selectors as possible out of this file to avoid potential conflicts with a users widgets css.

Reply:
It works with `pre[class*='language-']` which I've updated it to use